client/src/components/Navbar.tsx

```diff
@@ -21,35 +16,20 @@ const Navbar: React.FC = () => {
             <img
               src={logo}
               alt="FAO Logo"
-              className="h-8"
+              className="h-40"
+              // these are the setting from the on render logo
+              //class="w-16 md:w-28 h-auto my-2 md:my-4"
               onClick={takeToHome}
             />
```

Comment:
You don't need the `onClick` handler here as React has an in-built `Link` component. 

```
  {/* logo */}
          <div className="flex items-center space-x-4">
            <Link to="/"> {/* Wrap the logo in a Link component */}
              <img
                src={logo}
                alt="FAO Logo"
                className="h-40"
              />
            </Link>
          </div>
```

Reply:
Done Changed it and checked that it works